server/bundles/io.cloudbeaver.server/schema/service.sql.graphqls

```diff
@@ -356,4 +356,12 @@ extend type Mutation {
     # Returns SQLExecutionPlan
     asyncSqlExplainExecutionPlanResult(taskId: ID!): SQLExecutionPlan !
 
+    asyncSqlRowDataCount(
+        projectId: ID,
+        connectionId: ID!,
+        contextId: ID!,
+        resultsId: ID!
+    ): AsyncTaskInfo!
+
+    asyncSqlRowDataCountResult(taskId: ID!): Int
```

Comment:
please use string because js doesn't support long numbers

Reply:
also function must return not optional value or exception

Reply:
and maybe we need to rename it to `asyncResultSetRowDataCount` or something that lets us know that we can use it with any result and not only sql queries

Reply:
@Wroud  `sql` at the beginning is the name of module

Reply:
ok about sql
about Int than we need to change it in FSFile, but sessionExpireTime maybe can be kept as it is (if we sure that it's lower than 2^53 – 1)

Reply:
we can add new scalar like `Long` for that type of numbers

Reply:
added